kolibri/plugins/user_auth/assets/src/views/AuthBase.vue

```diff
@@ -42,8 +41,7 @@
             <h1
               v-if="themeConfig.signIn.showTitle"
               class="kolibri-title"
-              :class="$computedClass({ color: $themeBrand.primary.v_300 })"
-              :style="themeConfig.signIn.titleStyle"
+              :style="[ themeConfig.signIn.titleStyle, { color: $themeBrand.primary.v_300 }]"
```

Comment:
This would need to be reversed to allow the theme to overwrite the default style:

```suggestion
              :style="[ { color: $themeBrand.primary.v_300 }, themeConfig.signIn.titleStyle ]"
```

This is how it would have worked before since `style` takes precedence over `class`